polymesh_schema.json

```diff
@@ -1525,6 +1525,27 @@
                     }
                 ],
                 "type": "GranularCanTransferResult"
+            },
+            "balanceAt": {
+                "description": "Returns the identity-ticker balance at a checkpoint.",
+                "params": [
+                    {
+                        "name": "did",
+                        "type": "IdentityId",
+                        "isOptional": false
+                    },
+                    {
+                        "name": "ticker",
+                        "type": "Ticker",
+                        "isOptional": false
+                    },
+                    {
+                        "name": "checkpoint",
+                        "type": "CheckpointId",
+                        "isOptional": false
+                    }
+                ],
+                "type": "BalanceAtResult<Balance>"
```

Comment:
You need to also define what `BalanceAtResult` is. 
Also, we can't really have generic type params for custom structs in the schema so you'd need to define `BalanceAtResult` as a fixed type. See `CanTransferResult` for example.